### ISSUE TITLE
buggy accessibility code in HSlider

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#188]
This bit of buggy code was added to HSlider by @samreid, presumably as part of some accessibility experiment. It's buggy because it allows you to modify a slider that is disabled.

```js
    // Keyboard accessibility
    thumbNode.addInputListener( {
      keydown: function( event, trail ) {
        var keyCode = event.domEvent.keyCode;
        var delta = keyCode === Input.KEY_LEFT_ARROW || keyCode === Input.KEY_DOWN_ARROW ? -1 :
                    keyCode === Input.KEY_RIGHT_ARROW || keyCode === Input.KEY_UP_ARROW ? +1 :
                    0;
        var clampedValue = Util.clamp( valueProperty.get() + (range.max - range.min) * 0.1 * delta, range.min, range.max );
        valueProperty.set( options.constrainValue( clampedValue ) );
      }
    } );
``

**Comment by pixelzoom** [comment]
Fixed so that arrow buttons do nothing if slider is disabled.  The thumb can still receive focus while disabled.  I don't know if that's desired behavior, or how to address it if it's not.  Assigning to @samreid to do verify and handle any remaining work.

**Comment by samreid** [comment]
There is still an outstanding question of whether disabled components should be focusable.  Perhaps @emily-phet knows, or perhaps we need to research/investigate.

The solution from https://github.com/phetsims/sun/commit/e993654af986d86b79c4ea8b2b30f82a7f102e80 looks good.  I tested it in http://localhost:8080/beers-law-lab/beers-law-lab_en.html?ea&accessibility and verified that once the slider became disabled, the keyboard no longer moved the slider knob.  The slider track is non-focusable so it doesn't require a separate enabled check.

Closing.

**Comment by samreid** [comment]
By the way, here's an issue that pertains to whether disabled components should be focusable: https://github.com/phetsims/sun/issues/154
